CREDITS.ini

```diff
@@ -80,3 +81,7 @@ Name=Jens-Wolfhard Schicke-Uffmann
 GitHub=Drahflow
 BugFixes=true
 
+[Credit_12]
+Name=Emmett Lauren Garlitz - Some Little Sandy Rd, Elkview, West by GOD Virginia
+GitHub=Oll Em
+Quote=I have a nice cherry chess-top(Glass). "But remember, I NEVER played on it."
```

Comment:
@JoshVarga @redteam316 It seems the anti-1st amendment democrat party Hillary/NSA/MIB/CIA/FBI edited/hijacked this. Watch em. Only trust the marshalls with the black shirts or ask them personally.
Now get your chess on!